.github/workflows/dart.yml

```diff
@@ -44,11 +44,15 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install ObjectBox C-API
         run: ./install.sh
-      - run: ./tool/pub.sh get
-      - name: Generate ObjectBox models
-        run: ./tool/pub.sh run build_runner build
-      - name: Run tests
-        run: ./tool/pub.sh run test
+      - if: startsWith(matrix.os, 'windows')
+        run: mv lib/objectbox.dll objectbox/lib/
```

Comment:
Maybe run `install.sh` from the subfolder? Or maybe move it there or to the `tool` folder?

Reply:
yeah, I can run it from the subfolder... Won't move because we link from it in the installation instructions:

bash <(curl -s https://raw.githubusercontent.com/objectbox/objectbox-dart/main/install.sh)

Reply:
Oops, didn't look closely, thought it was the objectbox-c install script...